.github/workflows/ci.yml

```
@@ -0,0 +1,70 @@
+name: Backend CI
+
+on:
+  push:
+    branches-ignore:
+      - 'master'
+    tags-ignore:
+      - '**'
+    paths:
+      - '**.php'
+      - '**.yml'
+      - '**.yaml'
+      - 'app/schema/**.xml'
+      - 'composer.lock'
+      - 'phpcs.xml.dist'
+      - 'phpunit.xml.dist'
+      - 'psalm.xml'
+      - 'psalm-baseline.xml'
```

Why not just `on: push`, there is no psalm, yml/yaml files in this project?

---

No use running this on push to master, so wouldn't go with a simple `on: push`.
Do agree there are more paths listed then needed.
```suggestion
on:
  push:
    branches-ignore:
      - 'master'
    tags-ignore:
      - '**'
    paths:
      - '**.php'
      - 'composer.lock'
      - 'phpcs.xml.dist'
      - 'phpunit.xml.dist'
```